examples/graph_gps.py

```diff
@@ -65,8 +78,35 @@ def forward(self, x, pe, edge_index, edge_attr, batch):
         return self.mlp(x)
 
 
+class RedrawProjection(torch.nn.Module):
+    def __init__(self, model: torch.nn.Module,
+                 redraw_interval: Optional[int] = None):
+        super().__init__()
+        self.model = model
+        self.redraw_interval = redraw_interval
+        self.register_buffer('num_last_redraw', torch.tensor(0))
+
+    def redraw_projections(self):
```

Comment:
Looks like this is called before every batch. Why not just call it inside `PerformerAttention`?

Reply:
Right, I just moved it into the `PerformerAttention`